coverage/pom.xml

```diff
@@ -51,6 +51,12 @@
             <artifactId>party-routing-profile-example</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
+        <!-- maven plugins -->
```

Comment:
if you don't add tests, it doesn't make sense to add it to the coverage project

Reply:
Ok thanks, Added unit test for StringFormatter